masterlist.yaml

```
@@ -852,11 +852,46 @@ plugins:
       - 'BOM_OpenMW_plazas.esp'
       - 'BOM_pathgrid_reset.esp'
       - 'multipatch.esp'
-
+   ### BCOM - Optional files ###
   - name: 'BCOM_Suran Expansion.ESP'
     url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
     after: [ 'BOM_pathgrid_reset.esp' ]
 
+  - name: 'BOM_Stavs_MGO.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after: [ 'Beautiful cities of Morrowind.ESP' ]
+
+  - name: 'BOM - Taller Lighthouse.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after: [ 'BOM_pathgrid_reset.esp' ]
+
+  - name: 'BOM_White_Suran.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - 'BCOM_Suran Expansion.esp'
+      - 'Beautiful cities of Morrowind.ESP'
+      - 'Suran_Underworld_v3.esp'
+
+  - name: 'SOP_Canal_02.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after: [ 'Beautiful cities of Morrowind.ESP' ]
+
+  - name: 'SOP_RichPlaza_IziStyle.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    inc: [ 'Brevur of Balmora - Finally Some Good Statue Mod.ESP' ]
+    after: [ 'BOM_pathgrid_reset.esp' ]
```

I generally wouldn't put a newline between files from the same mod as well.

```suggestion
  - name: 'BCOM_Suran Expansion.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after: [ 'BOM_pathgrid_reset.esp' ]
  - name: 'BOM_Stavs_MGO.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after: [ 'Beautiful cities of Morrowind.ESP' ]
  - name: 'BOM - Taller Lighthouse.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after: [ 'BOM_pathgrid_reset.esp' ]
  - name: 'BOM_White_Suran.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - 'BCOM_Suran Expansion.esp'
      - 'Beautiful cities of Morrowind.ESP'
      - 'Suran_Underworld_v3.esp'
  - name: 'SOP_Canal_02.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after: [ 'Beautiful cities of Morrowind.ESP' ]
  - name: 'SOP_RichPlaza_IziStyle.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    inc: [ 'Brevur of Balmora - Finally Some Good Statue Mod.ESP' ]
    after: [ 'BOM_pathgrid_reset.esp' ]
```